src/terminal/Line.cpp

```diff
@@ -29,8 +29,13 @@ template <typename Cell>
 typename Line<Cell>::InflatedBuffer Line<Cell>::reflow(ColumnCount _newColumnCount)
 {
     using crispy::Comparison;
+    if (isTrivialBuffer())
+    {
+        auto buffer = TrivialBuffer();
+        buffer.displayWidth = _newColumnCount;
+        return {};
```

Comment:
Does this make sense?

Reply:
What confuses me is that two lines above you create a new variable of type `TrivialBuffer`, assign some value to `displayWidth` right after, but then don't use that variable. Did you probably mean this instead (two lines above)?

```cpp
auto& buffer = trivialBuffer();
```

(p.s.: Use `&` to denote that you are actually also want to use a reference, that might have maybe caught the typo.)

Reply:
Yes, I've made two typos there :)